pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -39,6 +39,7 @@
     "System.Threading.Tasks.Parallel": "4.0.1",
     "System.Threading.Thread": "4.0.0",
     "System.Threading.ThreadPool": "4.0.10",
+    "Microsoft.DiaSymReader.Native": "1.4.0-rc2",
```

Comment:
No we will need to have a stable version of this before we can ship a stable release of Microsoft.NETCore.App.